packages/react-chat/src/components/Proactive/index.tsx

```diff
@@ -27,7 +27,7 @@ export const ProactiveMessageContainer = styled('div', {
 export const ProactiveContainer = styled(tag('div'), {
   position: 'absolute',
   bottom: '100%',
-
+  width: 256,
```

Comment:
this is the issue with the proactive bubbles. embed mode doesn't even have this so it doesn't matter